src/ImageSharp/Dithering/ErrorDiffusion/IErrorDiffuser.cs

```diff
@@ -0,0 +1,34 @@
+﻿// <copyright file="IErrorDiffuser.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp.Dithering
+{
+    using System;
+
+    /// <summary>
+    /// Encapsulates properties and methods required to perfom diffused error dithering on an image.
+    /// </summary>
+    public interface IErrorDiffuser
+    {
+        /// <summary>
+        /// Gets the dithering matrix
+        /// </summary>
+        byte[][] Matrix { get; }
```

Comment:
Laziness 😈 I was reading from the samples I found that were using bytes. Ordered dithering requires bytes though since we are using to/.from bytes to reduce complexity for all pixel formats. Happy to switch to fast flat array. Throw one into this PR if you like?